src/components/SelectBox.js

```diff
@@ -60,10 +60,13 @@ class SelectBox extends React.Component {
   }
 
   render () {
-    const { id, disabled } = this.props
+    const { id, disabled, msg } = this.props
 
     const selectedItem = this.state.items.find(item => item.id === this.state.selected)
     const validationClass = this.getValidationClass()
+    const markAsDefault = () => {
```

Comment:
I don't think this needs to be a function....

You should be able to do:
```jsx
const MarkAsDefault = <i>{msg.defaultOption()}</i>
```

and then later just do:
```jsx
  {selectedItem && selecteItem.default && <MarkAsDefault />}
```